lib/datetime_helper/serialisers/active_model_serialiser_helper.rb

```diff
@@ -11,7 +11,7 @@ def enforce_zulu_time(attribute)
     # Enforces that a Date is serialized as iso8601 format
     def enforce_iso8601_date(attribute)
       define_method(attribute) do
-        object.send(attribute).iso8601 unless object.send(attribute).nil?
+        object.send(attribute).strftime('%F') unless object.send(attribute).nil?
```

Comment:
what's the difference? Both produce the same result but, to me anyway, using the `.iso8601` method is more descriptive.  With `strftime('%F')` a developer would probably have to go look that up to check.  I see no compelling reason to make this change.